win32/src/interlocked_hl_win32.c

```diff
@@ -8,11 +8,12 @@
 
 #include "azure_c_logging/xlogging.h"
 
+#include "azure_c_pal/interlocked.h"
```

Comment:
>interlocked [](start = 22, length = 11)

and sync.h?

Reply:
Or is that replacement coming in a second PR? Should also be a straight find and replace drop-in I think and it completely removes the need for windows.h

---
In reply to: [499933643](https://github.com/Azure/azure-c-pal/pull/39#discussion_r499933643) [](ancestors = 499933643)

Reply:
Will have to come in a separate PR.